man.Rmd

```diff
@@ -460,28 +548,39 @@ For example `readr::read_csv()` mentions what happens if there are any parsing p
 ## Submitting to CRAN
 
 For your initial CRAN submission, all functions must document their return value.
-This is not required for subsequent submissions, but it's still a good practice.
+While this may not be scrutinized in subsequent submissions, it's still a good practice.
 There's currently no way to check that you've documented the return value of every function (we're [working on it](https://github.com/r-lib/roxygen2/issues/1334)) which is why you'll notice some tidyverse functions lack output documentation.
+But we certainly aspire to provide this information across the board.
 :::
 
-## Examples {#sec-examples}
+## Examples {#sec-man-examples}
 
 Describing what a function does is great, but *showing* how it works is even better.
 That's the role of the `@examples` tag, which uses executable R code to demonstrate what a function can do.
 Unlike other parts of the documentation where we've focused mainly on what you should write, here we'll briefly give some content advice and then focus mainly on the mechanics.
-The mechanics of examples are complex because they must never error and they're run in four different situations:
+
+The main dilemma with examples is that you must jointly satisfy two requirements:
```

Comment:
I really want to make this fundamental dilemma with examples super concrete.

Reply:
I like it!